src/pages/General.vue

```diff
@@ -56,7 +56,7 @@ export default {
   async mounted() {
     ipcRenderer.on('settings-update', this.onSettingsUpdate);
     ipcRenderer.on('update-state', this.onUpdateState);
-    ipcRenderer.send('update-sate');
+    ipcRenderer.send('update-state');
```

Comment:
Does this fix a bug somewhere? Should we open an image for this alone, so we capture it better?

Reply:
This would have made the update state incorrect on startup.  But the update state is currently disabled (because we don't have the server set up yet), so the error wouldn't be visible.

I actually fixed this because the new checking means it actually failed (yay for type checking!), so I can't _not_ fix this in this PR :)